### DISCUSSION TITLE
world ball beginning position

### DISCUSSION THREAD

**Discussion by matusko89** [robotology/community#25] [Q&A] [answered]
Hi all,
I had somehow managed to overwrite the built-in ball starting position and now it is positioned under the table and I have to set it position every time I start the simulator. I tried to find something in the simConfig directory but with any luck. I assume there should be some .ini file for that but I am unable to find it.

Thanks 
Matus

**Comment by vtikha** [comment, marked as answer]
Hi @matusko89, the short answer is that, at the moment, the default sphere position cannot be configured in the simConfig or in any ini file. You will have to set its position or initialise the simulator's world as you please from your module, once you start the simulator. This has been noted as a future feature request for the simulator.

**Comment by lornat75** [comment]
@matusko89: how did you manage to change the ball position? in the code? Maybe you need to reset your repository and discard your local changes:

git reset --hard

be careful because you lose all your local changes.

**Comment by vtikha** [comment]
@lornat75 I think this is related to this issue #18

**Comment by matusko89** [comment]
@lornat75 I was messing around with yarp rpc /icubSim/world and trying if it is maybe possible to change color of ball. I managed to crash the simulator few times and the ball just happened to start render under the table...

**Comment by matusko89** [comment]
Nevertheless It is back to normal now but some .ini file would be nice for future.

Thanks

**Comment by vtikha** [comment]
@matusko89, make sure that you are not trying to place the ball within the table as that might create some issues due to various collisions. I should suggest to use the custom object creation instead of the default ones as you have more control on their properties. You can find everything [here](http://eris.liralab.it/wiki/Simulator_README#OBJECT_MANIPULATION:)
